data/src/main/java/io/gnosis/data/models/ChainInfo.kt

```diff
@@ -0,0 +1,30 @@
+package io.gnosis.data.models
+
+import com.squareup.moshi.Json
+import com.squareup.moshi.JsonClass
+
+
+@JsonClass(generateAdapter = true)
+data class ChainInfo(
+    @Json(name = "chainId") val chainId: Int,
+    @Json(name = "chainName") val chainName: String,
+    @Json(name = "rpcUrl") val rpcUrl: String,
+    @Json(name = "blockExplorerUrl") val blockExplorerUrl: String,
+    @Json(name = "nativeCurrency") val nativeCurrency: NativeCurrency,
+    @Json(name = "transactionService") val transactionService: String,
```

Comment:
we need transactionService url only to possibly show it on network details page?

Reply:
For the DTO model I wanted to include everything we get. No matter if we need it. I would reduce the data we store to what we need now or until the feature is delivered.